src/main/java/com/google/sps/data/TextAnalyser.java

```diff
@@ -26,10 +27,11 @@
 import com.google.cloud.language.v1.LanguageServiceClient;
 import com.google.cloud.language.v1.LanguageServiceSettings;
 import com.google.cloud.language.v1.Sentiment;
+import com.google.cloud.language.v1.Token;
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.Collections;
-import java.util.HashSet;
+import java.util.LinkedHashSet;
 import java.util.List;
 import java.util.Set;
 
```

Comment:
I know this was not touched by you in this PR, however Javadoc doesn't have trailing whiteline before the class. Also this Javadoc needs a rework

Reply:
Also Mara's

Reply:
Also, almost all (of not all) methods here are not self explanatory, you may want to add Javadoc to methods as well.